docs/io.cozy.apps.md

```diff
@@ -74,3 +74,15 @@ This is the truncated version of the official Cozy `drive` app.
   }
 }
 ```
+
+# Special `io.cozy.apps` doctypes 
+
+We have a few `magic folders` in the cozy ecosytem like `Admistrative` or `Photos`... In order to let the user rename these folders without having our app re-creating them everytime, we have used a few special doctypes to use them in the [`referenced_by` attribute](https://docs.cozy.io/en/cozy-stack/references-docs-in-vfs/#post-filesfile-idrelationshipsreferenced_by) . 
```

Comment:
```suggestion
Some folders are remarkable, like the 'Administrative' and the 'Photos' folders. You shouldn't address them by their path as it may vary depending on the user language and the user may have renamed them anyways.

To identify them, you can ask for a special [`referenced_by` attribute](https://docs.cozy.io/en/cozy-stack/references-docs-in-vfs/#post-filesfile-idrelationshipsreferenced_by).

For example, a request for all io.cozy.files referended by the doctype `io.cozy.apps` and the document id `administrative` will return you the io.cozy.files document of the folder 'Administrative' whatever its name or path.
```